src/Models/LoginSession.php

```diff
@@ -285,15 +294,24 @@ public function isCurrent(?HTTPRequest $request = null): bool
      */
     public static function getCurrentSessions(Member $member)
     {
-        $sessionLifetime = static::getSessionLifetime();
-        $maxAge = DBDatetime::now()->getTimestamp() - $sessionLifetime;
+        $maxAge = self::getMaxAge();
```

Comment:
```suggestion
        $maxAge = static::getMaxAge();
```
This isn't a final class - and in fact as a `DataObject` it could be replaced with dependency injection.